Source/Core/Core/DiscordPresence.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+// Discord Presence
+
+// Adds support for Discord Rich Presence in to Dolphin
+
+#pragma once
+#include <string>
```

Comment:
A newline should be before the include so it's separated from the header guard. e.g.

```cpp
#pragma once

#include <string>
```